From: Sean Finney <seanius@debian.org>
Date: Mon, 25 Feb 2013 11:40:18 +0100
Subject: nsca.cfg_nagios_vs_nagios

===================================================================
---
 sample-config/nsca.cfg.in | 12 +++++++-----
 1 file changed, 7 insertions(+), 5 deletions(-)

diff --git a/sample-config/nsca.cfg.in b/sample-config/nsca.cfg.in
index b866e38..4e68cf3 100644
--- a/sample-config/nsca.cfg.in
+++ b/sample-config/nsca.cfg.in
@@ -89,9 +89,12 @@ debug=0
 # COMMAND FILE
 # This is the location of the Nagios command file that the daemon
 # should write all service check results that it receives.
-
-command_file=@localstatedir@/rw/nagios.cmd
-
+# Note to debian users: icinga and nagios3 have
+# different default locations for this file.  this is the
+# default location for icinga
+# command_file=/var/lib/icinga/rw/icinga.cmd
+# and this is the default location for nagios3:
+command_file=/var/lib/nagios3/rw/nagios.cmd
 
 # ALTERNATE DUMP FILE
 # This is used to specify an alternate file the daemon should
@@ -103,8 +106,7 @@ command_file=@localstatedir@/rw/nagios.cmd
 # it starts Nagios.  Or you may simply choose to ignore any
 # check results received while Nagios was not running...
 
-alternate_dump_file=@localstatedir@/rw/nsca.dump
-
+alternate_dump_file=/var/run/nagios/nsca.dump
 
 
 # AGGREGATED WRITES OPTION
